src/JE.ApiValidation.FluentValidation.StructureMap/JE.ApiValidation.FluentValidation.StructureMap.v2.ncrunchproject

```diff
@@ -0,0 +1,26 @@
+﻿<ProjectConfiguration>
```

Comment:
Yes. This causes a problem, sometimes, in editors that don't recognise that the content is XML. At one point ncrunch itself had an issue around this.